File: CS/MVVMExpenses/Views/LoginView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Text;
using System.Linq;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using MVVMExpenses.ViewModels;

namespace MVVMExpenses.Views {
    public partial class LoginView : DevExpress.XtraEditors.XtraUserControl {
        public LoginView() {
            InitializeComponent();
            PasswordTextEdit.Properties.PasswordChar = '*';
        }

        protected override void OnLoad(System.EventArgs e) {
            base.OnLoad(e);
            var fluentAPI = mvvmContext1.OfType<LoginViewModel>();
            fluentAPI.SetObjectDataSourceBinding(userBindingSource,
                x => x.CurrentUser, x => x.Update());
            

            foreach(string item in mvvmContext1.GetViewModel<LoginViewModel>().LookUpUsers) 
                LoginTextEdit.Properties.Items.Add(item);
            fluentAPI.ViewModel.Init();
        }
    }
}

File: VB/MVVMExpenses/Views/LoginView.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Linq
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports MVVMExpenses.ViewModels
Imports DevExpress.Utils.MVVM.Services

Namespace MVVMExpenses.Common.Views
    Partial Public Class LoginView
        Inherits DevExpress.XtraEditors.XtraUserControl

        Public Sub New()
            InitializeComponent()
            PasswordTextEdit.Properties.PasswordChar = "*"c
        End Sub

        Protected Overrides Sub OnLoad(ByVal e As System.EventArgs)
            MyBase.OnLoad(e)
            Dim fluentAPI = mvvmContext1.OfType(Of LoginViewModel)()
            fluentAPI.SetObjectDataSourceBinding(UserBindingSource, Function(x) x.CurrentUser, Sub(x) x.Update())


            For Each item As String In mvvmContext1.GetViewModel(Of LoginViewModel)().LookUpUsers
                LoginTextEdit.Properties.Items.Add(item)
            Next item
            fluentAPI.ViewModel.Init()
        End Sub

    End Class
End Namespace
